CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
```

Comment:
in commit message: 
s/service work/service worker
s/and is saved we the other static file/and IT is saved WITH the other static fileS